tests/generate-and-approve-examples.php

```diff
@@ -12,5 +12,5 @@
     shell_exec("rm -rf {$expectedPath}");
 
     $generatedPath = Examples::generatedPath($example);
-    shell_exec("cp --recursive {$generatedPath} {$expectedPath}");
+    shell_exec("cp -R {$generatedPath} {$expectedPath}");
```

Comment:
Why change this? I generally prefer long-form options.

Reply:
cannot run on macos

![image](https://github.com/user-attachments/assets/353b60fb-de2e-434f-a8b5-12a6f82892ce)

Reply:
```suggestion
    # Using -R over --recursive for macOS compatibility
    shell_exec("cp -R {$generatedPath} {$expectedPath}");
```